gatsby-node.js

```diff
@@ -208,6 +211,11 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
            }
          }
        }
+       allIntegrations: allMdx(filter: { fields: { collection: { eq: "integrations" } } }) {
+        nodes {
+          slug
+        }
+      }
```

Comment:
Rebase the PR, and re-use the same query that is defined in the beginning i.e, `allPosts` and filter like any other collections like:
`node = > node.filter.collections === "integrations"`

Reply:
Okay!